packages/webpack/package.json

```diff
@@ -8,6 +8,7 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
+      "require": "./module.cjs",
```

Comment:
I suspect this is only required within the monorepo in stub mode rather than when built. Would you try removing the change to the packaging and running `pnpm build` before testing it?

Reply:
Hey @danielroe, I tried doing a pnpm build before adding the changes in webpack, but unfortunately, I'm getting the error message that says "No 'exports' main defined." after the build.